Acrolinx.Sidebar/Storage/RegistryAcrolinxStorage.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Acrolinx.Sdk.Sidebar.Util.Logging;
+using Microsoft.Win32;
+using System;
+
+namespace Acrolinx.Sdk.Sidebar.Storage
+{
+    public sealed class RegistryAcrolinxStorage : IAcrolinxStorage
+    {
+        private static readonly WeakReference instanceRef = new WeakReference(new RegistryAcrolinxStorage());
```

Comment:
why a WeakReference ?

Reply:
Instead of disposing the Singleton object we can use WeakRefrence. 

Object Lifetime:
1.	Strong reference will be created with RegistryStorage when sidebar object is instantiated.
2.	When sidebar object is disposed. The reference will become weak and will be available for garbage collection.
3.	If another sidebar object tries to obtain a reference to the RegistryStorage object, then the weak reference will be again made as a strong reference, if not garbage collected 

Scenario:  If multiple Word documents are opened and only one contains sidebar instance, if that document is closed AcrolinxStorage object will be available for Garbage collection.
Write up Courtesy: Abhijeet :)

Reply:
Removed use of WeakRefrence.